### ISSUE TITLE
Events crossing a day boundary

### ISSUE THREAD

**Issue by ohwgiles** [ohwgiles/focal#17]
Events that cross a day boundary are not drawn correctly. A proper solution should consider edge cases and performance concerns

**Comment by MoralCode** [comment]
do you have a screenshot of how they are being incorrectly drawn?

**Comment by ohwgiles** [comment]
No, just never implemented any logic to handle this case at all. Probably it would try to display it only on the start date, but with the end time of the end date instead of extending to midnight of the start date
